### PR TITLE
STUD-281: replace buttons with icons

### PR DESCRIPTION
**Pull request by yasfahm**
Changed the accept/deny buttons to icons
IMG_97A095246B0D-1

### CODE REVIEW

File: src/components/ParticipantListItem.js [hunk 1]
```diff
@@ -97,47 +107,32 @@ const ParticipantName = styled.Text `
...
 
-const ButtonText = styled.Text`
-  color: white;
-  font-family: ${theme.FONT_SEMIBOLD};
-  letter-spacing: ${theme.LETTER_SPACING_SMALL};
-  text-transform: uppercase
-  font-size: ${theme.FONT_SIZE_MEDIUM};
-`;
+const AcceptButton = styled.Image `
+  height: 20;
```
Comment (car12lin12) [inline src/components/ParticipantListItem.js]: I love the look of the button icons, however they're a bit tiny. Perhaps you can add the buttons to the next row and make them a bit larger?

Comment (yasfahm) [reply src/components/ParticipantListItem.js]: Stella didn’t like the original one to spill onto the next line :( so we opted for these icons.

Comment (yasfahm) [reply src/components/ParticipantListItem.js]: I changed the size of the buttons, see if it's better!

**Review by car12lin12 — APPROVED** [review]
looks good Yas! :rocket:

**Pull request merged**